cypress/e2e/gui/a11y.cy.js

```diff
@@ -0,0 +1,43 @@
+describe('EngageSphere', () => {
+  beforeEach(() => {
+    cy.mockCustomers().as('getCustomer')
+
+    cy.visit('/')
+    cy.injectAxe()
+  })
+
+  it('It finds no a11y issues in light mode in the customer table', () => {
```

Comment:
```suggestion
  it('finds no a11y issues in light mode in the customer table', () => {
```

Que tal remover a plavra _It_ visto que os testes estão dentro de um bloco `it`?

Reply:
Opa, esses passaram despercebido na hora de copiar o caso de teste